assessment/services/assessment.py

```diff
@@ -48,3 +50,130 @@ def create_assignment(request_data, user):
     validate_assignment(request_data)
     assignment = save_assignment_to_database(request_data, assessor)
     return assignment
+
+
+def validate_interactive_quiz(request_data):
+    if request_data.get('total_points') is None:
+        raise InvalidInteractiveQuizRegistration('Interactive Quiz should have total points')
+    if not isinstance(request_data.get('total_points'), int):
+        raise InvalidInteractiveQuizRegistration('Interactive Quiz total points must only be of type numeric')
+
+    if not request_data.get('duration_in_minutes'):
+        raise InvalidInteractiveQuizRegistration('Interactive Quiz should have a duration')
+    if not isinstance(request_data.get('duration_in_minutes'), int):
+        raise InvalidInteractiveQuizRegistration('Interactive Quiz duration must only be of type numeric')
+
+
+def save_answer_option_to_database(answer: dict, mc_question: MultipleChoiceQuestion):
+    content = answer.get('content')
+    correct = answer.get('correct')
+
+    answer_option = MultipleChoiceAnswerOption.objects.create(
```

Comment:
Hello, Indi. For better encapsulation, I believe it is better to place this inside a method in the MultipleChoiceQuestion.

Reply:
Thanks for the advice, Jo